### ISSUE TITLE
Use command line options instead of myconfig.py

### ISSUE THREAD

**Issue by liuml07** [shiqiezi/google-scholar-citations#10]
Command line options are preferred to a static myconfig file in terms of interactive running.

**Comment by shiqiezi** [comment]
[Refer to this project for usage](https://github.com/ckreibich/scholar.py/blob/master/scholar.py)

**Comment by shiqiezi** [comment]
@yilihong This patch was based on the offline discussion with @liuml07 I need 2nd opinion about his. Would you review & merge this PR if you find time? Thank you very much.

**Comment by yilihong** [comment]
@liuml07 @shiqiezi 

I think this is a great idea. I tested in Mac OS X and Ubuntu 14.04, Anaconda Python 2.7.12, on issue-10 branch. 

My comments:
1. Tested using `python main.py --request-interval=10 --should-download https://scholar.google.com/citations?user=VwQmUFQAAAAJ`, I didn't find a problem with the command line.
2. I encountered "UnicodeEncodeError" (something like this: http://stackoverflow.com/questions/9942594/unicodeencodeerror-ascii-codec-cant-encode-character-u-xa0-in-position-20) when writing some bibtex files (in the save_citation function), my temporary solution is to change line 130 to:

```
 with open(CITATION_FILENAME, "a+") as f:
        f.write(bib_entry.encode('utf-8'))
```
1. I am not sure if Google looks at requests patterns (obviously I used time.sleep(10) and was quickly blocked access), I usually do the following to specify the sleep function in crawlers. Perhaps it doesn't matter, if Google just checks number of requests per period of time.

```
from random import randint
time.sleep(randint(x,y))
```
